dev/bots/analyze-sample-code.dart

```diff
@@ -212,6 +213,11 @@ dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
+''');
+    analysisOptions.writeAsStringSync('''
+linter:
+  rules:
+    - unnecessary_const
```

Comment:
we probably want to apply many more lints, too... like maybe all the lints from the analysis_options file.

Reply:
Yes, this PR is just the first step ;) `unnecessary_new` will come quickly.